src/styles/index.scss

```diff
@@ -1,3 +1,140 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap';
```

Comment:
Unused `@import` statement. If you're not importing any files, this line should be removed to keep the code clean.